themes-default/slim/views/partials/showheader.mako

```diff
@@ -10,64 +10,47 @@
 <%namespace file="/inc_defs.mako" import="renderQualityPill"/>
 
 <div class="row">
-    <div id="showtitle" class="col-lg-12" data-showname="${show.name | h}">
+    <div id="showtitle" class="col-lg-12" :data-showname="show.title">
         <div>
-            <h1 class="title" data-indexer-name="${show.indexer_name}" data-series-id="${show.indexerid}" id="scene_exception_${show.indexerid}">
-                <app-link href="home/displayShow?indexername=${show.indexer_name}&seriesid=${show.indexerid}" class="snatchTitle">${show.name | h}</app-link>
+            <h1 class="title" :data-indexer-name="show.indexer" :data-series-id="show.id[show.indexer]" :id="'scene_exception_' + show.id[show.indexer]">
+                <app-link :href="'home/displayShow?indexername=' + show.indexer + '&seriesid=' + show.id[show.indexer]" class="snatchTitle">{{ show.title }}</app-link>
             </h1>
         </div>
 
-    % if action == 'snatchSelection':
-        <div id="show-specials-and-seasons" class="pull-right">
+        <div v-if="$route.name === 'snatchSelection'" id="show-specials-and-seasons" class="pull-right">
             <span class="h2footer display-specials">
-                <%include file="/partials/seasonEpisode.mako"/>
+                Manual search for:<br>
+                <app-link href="home/displayShow?indexername=${show.indexer_name}&seriesid=${show.series_id}" class="snatchTitle">{{ show.title }}</app-link> / Season {{ season }}<template v-if="episode"> Episode {{ episode }}</template>
             </span>
         </div>
-    % else:
-        % if season_results:
-            ##There is a special/season_0?##
-            % if int(season_results[-1]["season"]) == 0:
-                <% season_special = 1 %>
-            % else:
-                <% season_special = 0 %>
-            % endif
-            % if not app.DISPLAY_SHOW_SPECIALS and season_special:
-                <% lastSeason = season_results.pop(-1) %>
-            % endif
+        <template v-if="$route.name !== 'snatchSelection' && show.seasons && show.seasons.length >= 1">
```

Comment:
Unless I'm reading this wrong,
you can do without the `<template>` wrapper, and just put the `v-if` on the next element?